data-service/pom.xml

```diff
@@ -29,8 +29,6 @@
     <artifactId>data-service</artifactId>
 
     <properties>
-        <eureka.version>1.9.16</eureka.version>
-        <databind.version>2.9.8</databind.version>
         <spring-cloud-starter-ribbon.version>1.4.7.RELEASE</spring-cloud-starter-ribbon.version>
```

Comment:
As far as i can tell, this isn't used. Either remove or comment about it's usage

Reply:
removed